larray/core/session.py

```diff
@@ -216,6 +216,15 @@ def load(self, fname, names=None, engine='auto', display=False, **kwargs):
         >>> arr1, arr2, arr3 = s['arr1', 'arr2', 'arr3']  # doctest: +SKIP
         >>> # only if you know the order of arrays stored in session
         >>> arr1, arr2, arr3 = s.values()                 # doctest: +SKIP
+
+        Using .csv files (assuming the same session as above)
+
+        >>> s.save('data')                                # doctest: +SKIP
+        >>> s = Session()                                 # doctest: +SKIP
+        >>> # load all .csv files starting with "output" in the data directory
+        >>> s.load('data')                                # doctest: +SKIP
+        >>> # or equivalently in this case
+        >>> s.load('data/arr*.csv')                       # doctest: +SKIP
```

Comment:
Fair enough... I wanted to get away without writing new tests ;-), but you are right... I will add them...